Updated sort.rs to show `TopK`

Night-Amber3301 opened the pull request:
Closes: #7750

Replaced `SortExec: fetch={fetch}, expr=[{}]` with `SortExec: TopK(fetch={fetch}), expr=[{}]` in [sort.rs](https://github.com/apache/arrow-datafusion/blob/main/datafusion/physical-plan/src/sorts/sort.rs) file

alamb commented:
This looks really close @Night-Amber3301 -- thank you. I think some of the clippy errors will be resolved if you merge up from main. And it looks like there are just a few more tests to update

Let me know if you would like to some help and thanks agian!

Night-Amber3301 commented:
Yeah sure, thanks for consideration, I would love to work on it, will go through it in a while.

Night-Amber3301 commented:
I've merged it up from main, please check it out.

Thanks!

alamb commented:
> I've merged it up from main, please check it out.

It looks like somehow your changes were not pushed to github (the UI doesn't show any new changes)

Night-Amber3301 commented:
Any suggestions on why is this happening?

alamb commented:
> Any suggestions on why is this happening?

Perhaps you haven't pushed the change to github? What command are you using?

I would expect you would hav to do something like
```
git push
```

And make sure your "upstream" was set to your fork

liukun4515 commented:
> Any suggestions on why is this happening?

Can you provide your git log to here?

you can get the git log by 
```
git log
```

in your project workspace

Night-Amber3301 commented:
@liukun4515 Here are the logs:

@Night-Amber3301 ➜ /workspaces/arrow-datafusion (patch-1) $ git log
commit 29e2dc3387805a0caa29741e333795597c686675 (HEAD -> patch-1, origin/patch-1)
Author: Pratibhanu Jarngal <99665583+Night-Amber3301@users.noreply.github.com>
Date:   Fri Oct 6 10:38:20 2023 +0530

    Updated sort.rs
    
    solves: #7750
    
    Replaced `SortExec: fetch={fetch}, expr=[{}]` with 'SortExec: TopK(fetch={fetch}), expr=[{}]' in [sort.rs](https://github.com/apache/arrow-datafusion/blob/main/datafusion/physical-plan/src/sorts/sort.rs) file

Weijun-H commented on datafusion/physical-plan/src/sorts/sort.rs:
```diff
@@ -769,7 +769,7 @@ impl DisplayAs for SortExec {
                             f,
                             // TODO should this say topk?
-                            "SortExec: fetch={fetch}, expr=[{}]",
+                            "SortExec: TopK(fetch={fetch}), expr=[{}]",
```
If the sort does not use TopK, should we format it like `"SortExec: fetch={fetch}, expr=[{}]"`

alamb replied:
Yes, I think that is the intention

alamb commented:
Marking as draft as this PR is not waiting on feedback
